bin/p-s.js

```diff
@@ -55,7 +56,22 @@ runPackageScript({
 function getPSConfig() {
   var psConfigFilename = program.config ? resolve(process.cwd(), program.config) : findUp.sync('package-scripts.js')
   try {
-    return require(psConfigFilename)
+    // Require a module like babel-register
+    if (program.require) {
+      try {
+        require(program.require)
+      } catch (e) {
+        log.warn({
+          message: colors.yellow('Unable to preload ' + program.require),
+          ref: 'preload-module'
+        })
+      }
+    }
+    var config = require(psConfigFilename)
+    if (programm.resolve === 'babel-register') {
```

Comment:
The problem with babel-register is that you get the default export behind a module with a default property. This line fixes it but I think it's not really pretty :)